pg_search/src/api/index.rs

```diff
@@ -595,3 +615,80 @@ pub fn term_set(
 
     SearchQueryInput::TermSet { terms }
 }
+
+/// A type used whenever our builder functions require a fieldname.
+#[derive(
+    Debug,
+    Clone,
+    Ord,
+    Eq,
+    PartialOrd,
+    PartialEq,
+    Hash,
+    Serialize,
+    Deserialize,
+    PostgresType,
+    PostgresHash,
+    PostgresEq,
+    PostgresOrd,
```

Comment:
these three aren't necessary.  Creates a lot of schema churn for no actual value.